coderd/database/migrations/000037_jwt_licenses.up.sql

```diff
@@ -0,0 +1,11 @@
+-- No valid licenses should exist, but to be sure, drop all rows
+DELETE FROM licenses;
+ALTER TABLE licenses DROP COLUMN license;
+ALTER TABLE licenses RENAME COLUMN created_at to uploaded_at;
+ALTER TABLE licenses ADD COLUMN jwt text NOT NULL;
+-- prevent adding the same license more than once
+ALTER TABLE licenses ADD CONSTRAINT licenses_jwt_key UNIQUE (jwt);
+-- exp tracks the claim of the same name in the JWT, and we include it here so that we can easily
+-- query for licenses that have not yet expired.
+ALTER TABLE licenses ADD COLUMN exp timestamp with time zone NOT NULL;
```

Comment:
Might I suggest an alternative for SQL comments?

```suggestion
ALTER TABLE licenses ADD COLUMN exp timestamp with time zone NOT NULL;
COMMENT ON COLUMN licenses.exp IS 'Exp tracks the claim of the same name in the JWT, and we include it here so that we can easily query for licenses that have not yet expired.';
```

We have no precedent, but I find it nice when the reasoning can live with the code/db as part of `dump.sql` and when viewing the database tables.

Reply:
I was disappointed to learn that dump.sql doesn't contain these comments (we pass `--no-comments` to `pg_dump`), but if you look at with a traditional GUI tool, you'll see them.